examples/.interop/next-prisma-starter-migration/.env

```diff
@@ -0,0 +1,2 @@
+# Make sure to override these in deployment
```

Comment:
env should not be in git by default, maybe rename it to .env.example

Reply:
Thanks for spotting.

Just threw this up as showcase for an internal discussion, probably more to do before it's ready.